src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,56 @@
+.nav {
+
+  &__list {
+    display: flex;
+    margin: 0;
+    padding: 0;
+    list-style: none;
+  }
+
+  &__link {
+    position: relative;
+
+    display: flex;
+    align-items: center;
+    height: 60px;
+
+    text-transform: uppercase;
+    text-decoration: none;
+    @extend %normal-text;
+    line-height: 14px;
+    cursor: pointer;
+    color: $default-color;
+
+    &--is-active {
+      position: relative;
+      padding-left: 0;
+      color: $active-state-color;
+
+      &::after {
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+
+        height: 4px;
+        width: 100%;
```

Comment:
you can use here or `width: 100%; left: 0;` or `right: 0; left: 0;` you don`t need both